### DISCUSSION TITLE
Sequelize alpha-v7 version: want to use sql template tag

### DISCUSSION THREAD

**Discussion by ghost** [sequelize/sequelize#16711] [Help & Questions] [answered]
Hi,

I want to use sql template tag with sequelize.query() but I am not able to use
`import {sql, Sequelize} from ' @sequelize/core';

const sequelize = new Sequelize(process.env.LOCAL_DB_NAME, 
                                                     process.env.LOCAL_DB_USERNAME, 
                                                     process.env.LOCAL_DB_PASSWORD,
                                                     {
                                                         host: process.env.LOCAL_DB_HOST,
                                                         dialect: 'mysql'
                                 });
     const id = 5;
     sequelize.query(sql`SELECT * FROM users where ID = ${id}`)
`

But this is giving error ""sql" cannot be an object. Pass a string instead, and pass bind and replacement parameters through the "options" parameter'"

I am importing Sequelize, sql both from @sequelize/core

Here sql template tag is returning Literal

 & sequelize.query(sql) expecting first parameter as string.
Anyone here can help me.

**Comment by ephys** [comment, marked as answer]
This was fixed in the latest release https://github.com/sequelize/sequelize/releases/tag/v7.0.0-alpha.34
